learning_wavelets/training_scripts/exact_recon_unet_training.py

```diff
@@ -0,0 +1,70 @@
+from learning_wavelets.config import LOGS_DIR, CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_div2k, im_dataset_bsd500
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+import os.path as op
+import tensorflow as tf
+from tensorflow.keras.callbacks import TensorBoard, ModelCheckpoint
+import tensorflow_addons as tfa
+import time
+
+tf.random.set_seed(1)
+
+
+def train_unet(noise_std_train=(0, 55), noise_std_val=30, n_samples=2000, source='bsd500', base_n_filters=4, batch_size=8, n_epochs=50):
```

Comment:
```suggestion
def train_unet(noise_std_train=(0, 55), noise_std_val=30, n_samples=None, source='bsd500', base_n_filters=4, batch_size=8, n_epochs=50):
```

The default for `n_samples` should be `None` because this means that we don't limit the dataset